src/main/java/org/example/Services/CustomerServices.java

```diff
@@ -27,27 +28,54 @@ public void depositMoney(String customerName, int depositAmount) throws IOExcept
         }
     }
 
+    public void makeTransfer(String nameSenderCustomer, String nameReceiverCustomer, int amountToSend) throws SQLException, IOException {
+        int senderreturnedId = dbcontroller.findCustomerByUsername(nameSenderCustomer);
+        int receiverreturnedId = dbcontroller.findCustomerByUsername(nameReceiverCustomer);
 
-    public boolean sendMoneyToSomeone(String nameSenderCustomer, String nameReceiverCustomer, int amountToSend) throws IOException, SQLException {
+        boolean sendmoney = sendMoneyToSomeone(senderreturnedId, amountToSend);
+        if (sendmoney){
+            receiveMoney(receiverreturnedId, amountToSend);
+            updateDbAfterTransfer(senderreturnedId, nameSenderCustomer, receiverreturnedId, nameReceiverCustomer, amountToSend);
+            System.out.println("Transfer is done");
+        }
+    }
+
+ // what if funds insufficient? add error handlers

```

Comment:
Small trick you may not know:
```suggestion
 // TODO: what if funds insufficient? add error handlers
```
A `TODO:` as a comment prefix alerts your IDE to treat it as a TODO comment, which this one is. It is perfectly fine to leave these in unfinished parts of code. Don't overdo it, of course, but they are fine to use anyway.